dagger/config/setup.go

```diff
@@ -8,11 +8,12 @@ import (
 	"path/filepath"
 	"strings"
 
+	"slices"
```

Comment:
Do we use any convention for import orderings? Is it just stdlib first and then external?

Reply:
I let gopls order the imports however it wants, although it ignores blank spaces between blocks of imports which sometimes requires manually removing them to make it reorder everything.